src/providers/auth_backend.ts

```diff
@@ -77,7 +78,8 @@ export interface FirebaseAuthState {
 
 export function authDataToAuthState(authData: FirebaseAuthDataAllProviders): FirebaseAuthState {
   let {auth, uid, provider, github, twitter, facebook, google, password, anonymous} = authData;
-  let authState: FirebaseAuthState = {auth, uid, provider: null};
+  let authState: FirebaseAuthState = { auth, uid, provider: null };
+  authState.expires = authData.expires;
```

Comment:
Please move into the original object assignment:

`let authState: FirebaseAuthState = { auth, uid, provider: null, expires: authData.expires };`